### ISSUE TITLE
AM-Softmax Implementation

### ISSUE THREAD

**Issue by pribadihcr** [opencv/openvino_training_extensions#311]
Is there AM-Softmax implementation in this paper https://arxiv.org/pdf/2003.07618.pdf ?. Thanks

**Comment by AlexanderDokuchaev** [comment]
Yes, it's there.
https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/person_reidentification/engine/losses/am_softmax.py
